podvm/Dockerfile.podvm_binaries

```diff
@@ -36,7 +36,7 @@ RUN ./cloud-api-adaptor/podvm/hack/cross-build-extras.sh
 RUN cd cloud-api-adaptor/podvm && \
      make binaries
 
-RUN tar czvf /podvm-binaries.tar.gz -C /src/cloud-api-adaptor/podvm/files usr/
+RUN tar czvf /podvm-binaries.tar.gz -C /src/cloud-api-adaptor/podvm/files usr/ etc/agent-config.toml
```

Comment:
I think it's better to copy the entire `/etc` folder in the podvm-binaries.tar.gz.
There are two modifications we do currently. One is the agent-config.toml and another is for tls certificates if required.

Reply:
@bpradipt OK, I see.

Could you elaborate this mechanism? I am wondering why we don't need to copy `etc/systemd` here. The built image is functional, so I miss something here.

Reply:
because when we set AA_KBC modify the config file in-place, hence it would require you to set the AA_KBC both when building the binaries container and when building the podvm, in practice we set it only when creating the binaries container

Reply:
@yoheiueda as mentioned earlier, we do modify systemd startup if there are tls certificates to be used. 
I think it's better to copy /etc in the podvm-binaries.tar.gz and extract it when building the podvm image.

cc @snir911

Reply:
done